9.tcga-classify/README.md

```diff
@@ -34,6 +34,14 @@ We constructed supervised machine learning tasks to distinguish each cancer-type
 We selected the [top 50 most mutated genes](https://github.com/greenelab/BioBombe/blob/master/9.tcga-classify/top-50-pancanatlas-mutations.ipynb) in TCGA to predict.
 Using non silent mutation and copy number status, we predicted the alteration status of these genes in the included cancer types.
 
+## Ensemble Models
```

Comment:
Any special reason for using the top 50?

Reply:
nice round number that would capture bona fide oncogenes and also a bunch of genes with high passenger mutation burden